src/getter.jl

```diff
@@ -0,0 +1,443 @@
+
+Depending on the coordinate system, this might be an equivalent but faster version of `cos(phi(lv))`.
+
+# Example
+If ``(x, y, z, t)`` is a four-momentum, this is equivalent to ``x / \\sqrt{x^2 + y^2}``.
+
+# See Also
+- [`LorentzVectorBase.phi`](@ref): for the azimuthal angle itself.
+- [`LorentzVectorBase.sin_phi`](@ref): for a faster version of the sine of the azimuthal angle.
+"""
+cos_phi
+
+"""
+    sin_phi(lv)
+
+Return the sine of the azimuthal angle (``\\phi``) of the Lorentz-vector-like `lv`.
+
+Depending on the coordinate system, this might be an equivalent but faster version of `sin(phi(lv))`.
+
+# Example
+If ``(x, y, z, t)`` is a four-momentum, this is equivalent to ``y / \\sqrt{x^2 + y^2}``.
+
+# See Also
+- [`LorentzVectorBase.phi`](@ref): for the azimuthal angle itself.
+- [`LorentzVectorBase.cos_phi`](@ref): for a faster version of the cosine of the azimuthal angle.
+"""
+sin_phi
+
+### Masses
+"""
+    mass2(lv)
+
+Return the squared invariant mass of the Lorentz-vector-like `lv`, computed as the Minkowski inner product of the four-vector components with itself.
+
+
+# Example
+For a four-momentum `(px, py, pz, E)`, this function returns `E² - (px² + py² + pz²)`.
+
+# See Also
+- [`mass`](@ref): For the invariant mass, i.e., the square root of this value.
+
+"""
+mass2
+
+"""
+    mass(lv)
+
+Return the invariant mass  of the Lorentz-vector-like `lv`, computed as the square root of the Minkowski inner product of the four-vector components with itself.
+
+
+# Example
+For a four-momentum `(px, py, pz, E)`, this function returns `sqrt(E² - (px² + py² + pz²))`.
+
+# Notes
+- If the squared invariant mass `m²` is negative, this function returns `-sqrt(-m²)` to ensure a real result. This can happen in certain unphysical cases where the energy component is smaller than the spatial momentum magnitude.
+
+# See Also
+- [`mass2`](@ref): For the squared invariant mass calculation.
+
+"""
+mass
+
+### Boost Parameter
+"""
+    boost_beta(lv)
+
+Return the magnitude of the velocity (``\\beta``) of a particle as a fraction of the speed of light of the Lorentz-vector-like `lv`.
+
+
+# Example
+For a four-momentum `(px, py, pz, E)`, this function returns `sqrt(px² + py² + pz²) / E`.
+
+# Throws
+- `ArgumentError` if the time (energy) component is zero while the spatial components are non-zero, as this results in an undefined velocity.
+
+# Notes
+- If both the time (energy) and spatial components are zero, the function returns zero (``\\beta = 0``), as this represents a stationary object with no motion.
+
+# See Also
+- [`boost_gamma`](@ref): For the relativistic gamma factor, which depends on ``\\beta``.
+"""
+boost_beta
+
+"""
+    boost_gamma(lv)
+
+Return the relativistic gamma factor (``\\gamma``) for the Lorentz-vector-like `lv`.
+
+
+# Example
+For a four-momentum `(px, py, pz, E)` with velocity `β`, this function returns `1 / sqrt(1 - β²)`.
+
+# See Also
+- [`boost_beta`](@ref): For the velocity `β`, which is used to compute `γ`.
```

Comment:
nit: should probably either use or avoid using ISO characters (β, γ, ²) everywhere consistently.

Reply:
done!